src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST" onsubmit="onSubmit()"
+    >
```

Comment:
If there are more than two attributes, each should start on separate line.
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="POST"
      onsubmit="onSubmit()"
    >
```